block/mempool_test.go

```diff
@@ -324,7 +270,7 @@ func (suite *BlockBusterTestSuite) TestNewMempool() {
 			lanes,
 			fetcher,
 		)
-		suite.Require().Error(err)
+		suite.Require().NoError(err)
```

Comment:
why is this behavior different now? just curious

Reply:
we dont enforce that a default lane is provided. up to the app developer to do that